lambdas/check_new_areas/function.py

```diff
@@ -1,20 +1,138 @@
-from botocore.exceptions import ClientError
-from summary_analysis_batch.utils import slack_webhook, s3_directory_exists
-from summary_analysis_batch.summary_batch_job import get_summary_analysis_step, submit_summary_batch_job
+import io
+import json
+import os
+from contextlib import contextmanager
+from datetime import datetime
+from typing import Any, Dict, Iterator, List, Optional
+
 import boto3
-import datetime
+import requests
+from requests import Response
+from shapely.wkb import dumps
+from shapely.geometry import shape, Polygon
+
+# environment should be set via environment variable. This can be done when deploying the lambda function.
```

Comment:
Setting the environment settings here via system variable instead of passing it as function parameter.
When deploying lambda function, you can set the sys variables